aws/aws.go

```diff
@@ -17,7 +17,7 @@ import (
 func GetAllRegions() []string {
 	// chinese and government regions are not accessible with regular accounts
 	reservedRegions := []string{
-		"cn-north-1", "cn-northwest-1", "us-gov-west-1",
+		"cn-north-1", "cn-northwest-1", "us-gov-west-1", "us-gov-east-1",
```

Comment:
Should this be configurable? That is, other cloud-nuke users may use these regions... Should there be some CLI flag to specify which ones to use or ignore? If so, it's not a high priority change, so feel free to file an issue for it so we can come back to it later.

Reply:
I think that's a separate feature from this. This just concerns the regions in China and GovCloud that you need special credentials to access. We already have an `exclude-region` CLI option that can be used with the regular AWS regions